extensions/wikia/AdEngine/js/lookup/rubicon/rubiconFastlane.js

```diff
@@ -198,7 +198,7 @@ define('ext.wikia.adEngine.lookup.rubicon.rubiconFastlane', [
 		return price;
 	}
 
-	function getSlotParams(slotName) {
+	function getSlotParams(slotName, floorPrice) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has a complexity of 11 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)